module-1/lab-bag-of-words/your-code/main.ipynb

```diff
@@ -9,11 +9,11 @@
   },
```

Comment:
Short, clean and readable!! 
PD: I would suggest you to include comments on your code explaining what each chunk of code expected behaviour is. In other words, what the ouptput is.